datacube_stats/output_drivers.py

```diff
@@ -643,6 +643,7 @@ def _handle_to_path(self, file_handle) -> Path:
         return Path(file_handle.filepath())
 
     def write_chunk(self, prod_name, chunk, result):
+        print(chunk)
```

Comment:
Printing to `stdout` is confusing... if you want, you could log it. But more likely you just want to delete this line.

Reply:
I remember i deleted it.. let me do again.